ci-operator/step-registry/rosa/nodes/hypershift-infra/rosa-nodes-hypershift-infra-commands.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+log(){
+    echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") " "${*}\033[0m"
+}
+
+function getDesiredInfraCount() {
+  echo "Find desired Infra node count"
+  desired_infra_count=$(ocm get /api/clusters_mgmt/v1/clusters/"$CLUSTER_ID"/node_pools | jq -r '[.items[] | select(.labels != null) | select(.labels."node-role.kubernetes.io/infra").replicas] | add')
+  echo "Desired Infra node count: $desired_infra_count"
+}
```

Comment:
I believe the recent job failure is due to the lack of ocm binary.
I think we can tackle this with `rosa list machinepool -c $CLUSTER_ID -o json`

Reply:
yes I tried, but weirdly hypershift cluster machinepool is not outputting in json thats why I had this.